packages/plugin-core/src/test/suite-integ/C_autoCompleter.test.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * File has C_ prefix to move it down in the list of tests.
+ * If C_ is removed it bubbles up to the top of the tests and for some reason
```

Comment:
Will merge it so we can get feedback on the tab completion. Since circular dependencies are becoming more of an issue, can you spend a few more cycles here to see if you can diagnose it? (eg. 30min time box and writeup of what you found for next person to take over?)

Reply:
Fixed this particular instance https://github.com/dendronhq/dendron/pull/1810